src/redux/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import booksReducer from './books/books';
```

Comment:
- [ ] [REQUIRED] Kindly add your `categoriesReducer` because it is currently missing, also, you do not have `categories.js` in your redux which should have a reducer and an action just like the `book.js`.